### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by larryonoff**

### CODE REVIEW

**Comment by ra1028** [comment]
I described about `setData` closure on the READM as follows:
`Don't forget to update synchronously the data referenced by the data-source in setData closure, as the differences is applied in stages`

I think that your changed part does not matter.

**Comment by ra1028** [comment]
Do you think about? @Sameesunkaria

**Comment by Sameesunkaria** [comment]
I guess we don't need the ⚠️, rest is fine.

**Comment by ra1028** [comment]
agreed

File: README.md [hunk 1]
```diff
@@ -45,10 +45,10 @@ Sample app is imitated from <a href="https://github.com/muukii/DataSources">Data
 The algorithm used in DifferenceKit is optimized based on the Paul Heckel's algorithm.  
 See also his paper ["A technique for isolating differences between files"](https://dl.acm.org/citation.cfm?id=359467) released in 1978.  
 [RxDataSources](https://github.com/RxSwiftCommunity/RxDataSources) and [IGListKit](https://github.com/Instagram/IGListKit) are also implemented based on his algorithm.  
-This allows all types of differences to be computed in linear time **O(n)**.  
+This allows all types of differences to be computed in linear time **O(n)**.
 
-However, in `performBatchUpdates` of UITableView and UICollectionView, there are combinations of operations that cause crash when applied simultaneously.  
-To solve this problem, DifferenceKit takes an approach of split the set of differences at the minimal stages that can be perform batch-updates with no crashes.
+⚠️ However, in `performBatchUpdates` of `UITableView` and `UICollectionView`, there are combinations of operations that cause crash when applied simultaneously.
```
Comment (ra1028) [inline README.md]: Please remove this ⚠️.

Comment (ra1028) [reply README.md]: Also, revert the last two spaces for line breaking.

**Comment by ra1028** [comment]
Please squash your commits into one commit after fixed.

**Comment by larryonoff** [comment]
In my opinion there should be accent about using reload and don't merge with rest of the text.

**Comment by larryonoff** [comment]
The idea of the ⚠️ is from https://github.com/OAuthSwift/OAuthSwift

**Comment by Sameesunkaria** [comment]
IMHO, @larryonoff adding a ⚠️ to that line is calling attention to an implementation detail. It's there for someone to go through, but doesn't require their immediate attention.

**Comment by larryonoff** [comment]
I would better leave it as it is. It's impossible to understand what and how other people accept improvements / changes in documentation

**Comment by ra1028** [comment]
Anyway, LGTM
Thanks @larryonoff

**Comment by larryonoff** [comment]
thanks for the great library

**Pull request merged**